pcsx2/GS/Renderers/HW/GSTextureCache.cpp

```diff
@@ -99,19 +99,15 @@ void GSTextureCache::RemoveAll()
 
 GSTextureCache::Source* GSTextureCache::LookupDepthSource(const GIFRegTEX0& TEX0, const GIFRegTEXA& TEXA, const GSVector4i& r, bool palette)
 {
-	if (!m_can_convert_depth)
+	if (m_renderer->m_game.title == CRC::SVCChaos)
+	{
+		// SVCChaos black screen on main menu, regardless of depth enabled or disabled.
+		return LookupSource(TEX0, TEXA, r, false);
```

Comment:
Is this really needed? Having hardcodded crc hacks is really bad. I guess a patch would be a better solution to this.

Reply:
You're welcome to submit a proper fix.

Reply:
I don't have the game, nor the knowledge of how to fix this, I just asked if it is still needed, which is apparently the case if you said so. Why this can't be moved to the crc hack database instead?